imageio/core/functional_api.py

```diff
@@ -0,0 +1,13 @@
+from .imopen import imopen
+import numpy as np
+
+
+def imread(uri, *, index=None, plugin=None, **kwargs):
+    with imopen(uri, **kwargs) as img_file:
```

Comment:
Nice to see how short this is! The `plugin` arg should be fed into `imopen` tho.

Reply:
Good catch, I have forgotten to route the plugin; I will add it.